.changeset/thin-ravens-begin.md

```diff
@@ -0,0 +1,5 @@
+---
+"saleor-app-taxes": minor
+---
+
+Adds `resolveOptionalOrThrow` util that throws when not able to resolve an optional value. Now, when critical values from external APIs are not found, the app will crash instead of falling back to 0s.
```

Comment:
I think its better to omit implementation details (users dont care about function name) - and focus on changed behavior, so maybe something like:

```
Changed how missing values from external APIs are handled. Before, missing values were falling back to `0`. Now, app will throw error, so webhook will fail and taxes will not be calculated
```